server/tests/controller/article.spec.js

```diff
@@ -6,32 +6,39 @@ import sinonChai from 'sinon-chai';
 import app from '<server>/app';
 import models from '<server>/models';
 import { article } from '<fixtures>/article';
+import { newCategoryData } from '<fixtures>/category';
 import createArticle from '<controllers>/article';
 import signupUser from '<test>/helpers/utils';
 
 const { expect } = chai;
 chai.use(chaiHttp);
 chai.use(sinonChai);
 
-const { Article, sequelize } = models;
-
+const { Article, Category, sequelize } = models;
+let newArticle;
 before(async () => {
   await sequelize.sync({ force: true });
+  const articleCategory = newCategoryData({ name: 'technology' });
+  const category = await Category.create(articleCategory);
+  const { id } = category;
+  article.categoryId = id;
+  newArticle = article;
```

Comment:
write a helper function to create a new article or seed the database before the tests. What you are doing here will not scale